examples/integration_testing.rs

```diff
@@ -80,9 +80,12 @@ impl Middleware<ServerData> for JsonPost {
 
 #[tokio::main]
 async fn main() {
-    let port = env::var("PORT").map(|s| s.parse().unwrap()).unwrap_or(3000);
-    let address = &format!("0.0.0.0:{}", port);
-    let database = vec![];
+    let port = env::var("PORT").map(|s| s.parse().unwrap()).unwrap_or(0);
+    let address = &format!("127.0.0.1:{}", port);
+    let database = match env::var("ALT_USERS") {
+	Err(_) => vec![],
+	Ok(_) => vec!["bors".into(), "homu".into(), "highfive".into()],
+    };
```

Comment:
I like these changes.  Aside: I want to explore `start_server` a little more; that `Database` trait looks interesting.